plugin/src/main/scala/com/typesafe/genjavadoc/BaseComments.scala

```diff
@@ -58,7 +60,7 @@ trait BaseComments { this: TransformCake ⇒
       new Comment(pos, l2.reverse map htmlEntity)
     }
     private def preLine(line: String): String =
-      line.replace("@", "&#64;").replace("<", "&lt;").replace(">", "&gt;")
+      line.replace("@", "&#64;").replace("<", "&lt;").replace(">", "&gt;").replace("->", "-&gt;")
```

Comment:
Good point, my addition is obviously not used, I think the problem was in java code in akka, so I changed that instead. I will remove this.